node/lib/DeviceAccessor.ts

```diff
@@ -341,7 +347,15 @@ export class DeviceAccessor {
 
     private static capitalize(propertyName: string) {
         if (propertyName.length > 1) {
-            return propertyName.substr(0, 1).toUpperCase() + propertyName.substr(1);
+            return propertyName[0].toUpperCase() + propertyName.substr(1);
+        }
+
+        return propertyName;
+    }
+
+    private static uncapitalize(propertyName: string) {
+        if (propertyName.length > 1) {
+            return propertyName[0].toLowerCase() + propertyName.substr(1);
         }
```

Comment:
I had this doubt. Doesn't capitalize/uncapitalize make fully upper/lower case words, instead of Pascal/Camel case, respectively ?